packages/gestalt/src/Dropdown.js

```diff
@@ -198,12 +199,12 @@ export default function Dropdown({
 }
 
 Dropdown.propTypes = {
-  // $FlowFixMe[signature-verification-failure] flow 0.135.0 upgrade
-  anchor: PropTypes.shape({
+  // $FlowFixMe[incompatible-cast]
```

Comment:
I tried to fix all these and was mostly shut down, but at least updated to more specific suppressions